sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,282 @@
+- Implemented `mem_advise` reset and managed memory checks for CUDA backend. [fe18839c]
+- Added concurrent memory check to `mem_advise` on CUDA backend. [33746d8d]
+- Enabled multiple HIP streams per SYCL queue. [e0c40a9f]
+- Implemented lazy mechanism of setting context for default-constructed events. [ed92c4ca]
+
+### SYCL Compiler
+
+- Increased max `_Bitint` size to 4096 for FPGA target. [db5f72a8] [3f06cad0]
+- Removed deprecation message for `[[intel::disable_loop_pipelining]]` attribute. [07201f56]
+- Allowed `__builtin_assume_aligned` to be called from device code. [24937eac]
+- Improved link step performance when `per_kernel` device code split is used. [84de9d6d]
+- Added support for `SYCL_EXTERNAL` on `device_global` variables. [8b958f67]
+- Updated `__builtin_intel_fpga_mem` to accept more parameters. [231338dc]
+- Updated `ivdep` attribute to allow `safelen = 0`. [558b3ba4]
+- Improved linking with `sycl.lib` on Windows. [404d2816]
+- Implemented more diagnostics about incorrect `device_global` usages. [12657218]
+- Improved library resolution for `libsycl.so`. [4ce19d69]
+- Improved diagnostics when linkind with mismatched objects. [0e0202ee]
+- Added a warning for floating-point size changes after implicit conversions. [e4f5d55f]
+- Made `invoke_simd` convert its argument to appropriate types. [038764fd]
+
+### Documentation
+
+- Removed explicit `cl` namespace references. [433ea5c7]
+- Added a short guildeline on using CMake with SYCL compiler. [fa603c3e]
+
+## Bug fixes
+
+### SYCL Library
+
+- Fixed an issue with `queue` shortcuts with offset parameter [f4f83d95]
+- Fixed `sycl::get_pointer_device` free function for cases with descendent
+  devices. [26d5d98b]
+- Fixed memory leak happening when programs/kernel bundles are linked. [980677d9]
+- Fixed USM free for descendent devices. [c49d4944]
+- Fixed `accessor`'s CTAD for `g++` host compiler. [57aabe7e]
+- Fixed template argument propagation in `accessor` subscript operator. [22e3fc56]
+- Fixed issues with multiple buffer reductions. [a0a4d721]
+- Fixed an issue with SYCL math built-ins when GCC < 11.1 is used as a host
+  compiler. [c786894f]
+- Fixed an issue with SYCL math built-ins not accepting pointers to `half`. [e2861665]
+- Fixed an issues with `reduction`s when MSVC is used as host compiler. [94c4b80a]
+- Fixed a compilation issue with fully specialized `sycl::span`. [2b50820b]
+- Fixed a crash in Level Zero PI plugins caused by specialization constants not
+  being used on device side, but present in a program. [9500875f]
+- Fixed event leak in Level Zero plugin. [6d04aa64]
+- Fixed an issue with sub-sub-devices in Level Zero plugin. [4b1b01bd]
+- Fixed an issue in OpenCL PI plugin when creating a `queue` with properties
+  unsupported by OpenCL. [c0d639e0]
+- Fixed an issue with incorrect `half` conversion on ESIMD emulator. [6143e55a]
+- Fixed a compilation issue with `abs` ESIMD function. [c72a85dd]
+- Fixed C++14/C++20 compatibilitye issues. [12ac4c36]
```

Comment:
```suggestion
- Fixed C++14/C++20 compatibility issues. [12ac4c36]
```

Reply:
This one got fixed in b27e48ded194c438115fd35457ddfb0dbf99b816